Bottom: f5c411cb61d9b787a921f3421515a434af2e5689
Top:    03a2ef329401f5900e8b94e7e3db1b3eb3e26d9f
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-06-07 16:06:51 +0300

Use designated initializer for struct list_head in list.h


---

diff --git a/include/linux/list.h b/include/linux/list.h
index 4b129df4d46b..11847c96571f 100644
--- a/include/linux/list.h
+++ b/include/linux/list.h
@@ -18,7 +18,10 @@
  * using the generic single-entry routines.
  */
 
-#define LIST_HEAD_INIT(name) { &(name), &(name) }
+#define LIST_HEAD_INIT(name) {	\
+	.next = &(name),	\
+	.prev = &(name),	\
+}
 
 #define LIST_HEAD(name) \
 	struct list_head name = LIST_HEAD_INIT(name)
